dockerfiles/ci/Dockerfile_buster

```diff
@@ -0,0 +1,444 @@
+ARG VERSION=v7_3
```

Comment:
Do we need to [include the license](https://github.com/docker-library/php/blob/master/LICENSE) with these files? :)

Reply:
Good point, we have to start adding those elsewher. I think the original file didn't have the copyright notice. But we need to mark the copyrights somewhere. I'll add that to the PR

Reply:
https://github.com/DataDog/dd-trace-php/pull/516